src/custom_node_interface.hpp

```diff
@@ -0,0 +1,33 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+struct CustomNodeTensor {
+    const char* name;
+    unsigned char* data;
+    int dataLength;
+    int* dims;
+    int dimsLength;
+    int precision;
+};
+
+struct CustomNodeParam {
+    const char *key, *value;
+};
```

Comment:
Shouldn't be everything extern C?

Reply:
`extern "C"` disabled symbol name mangling. From my understanding, we shouldn't care about struct name mangling, all wee need is to be sure that structs used in OVMS and shared libraries are of same size.

Reply:
To make sure structs defined in OVMS and shared libraries are of the same size we need to compile it on the same architecture (x64 in this case)